pretend.py

```diff
@@ -107,9 +108,18 @@ def __repr__(self):
         return "<call(%s%s%s)>" % (args, comma, kwargs)
 
 
-def call_recorder(func):
+def call_recorder(func, real_func=None, real_method=None):
```

Comment:
call_recorder is usable as a decorator, which will still work with this PR, but won’t support passing the new parameters.
Could the function support usage as a decorator factory?  usage:

```python
@pretend.call_recorder(real_func=my_real_function)
def fake_func(a, b):
    return "canned result"
```

Reply:
Huh. I had no idea people were using it as a decorator. (Frankly, I didn't realize people used this library at all)

Reply:
I find its design elegant compared to mock, even though one of the big pain point of stdlib mock is removed with spec/autospec.  I use pretend in some personal and profressional projects!